ttl/ttlworker/worker.go

```diff
@@ -122,6 +124,9 @@ func (w *baseWorker) Send() chan<- interface{} {
 func (w *baseWorker) loop() {
 	var err error
 	defer func() {
+		if r := recover(); r != nil {
+			logutil.BgLogger().Info("ttl worker panic", zap.Any("message", r))
```

Comment:
In general, we used to call it ```recover```.

```suggestion
			logutil.BgLogger().Info("ttl worker panic", zap.Any("recover", r))
```

Reply:
Done.